user-service/controller/user-controller.js

```diff
@@ -10,6 +10,15 @@ import bcrypt from 'bcrypt';
 import jwt from 'jsonwebtoken';
 import 'dotenv/config'
 
+export async function getUser(req, res) {
+    try {
+        const user = req.user;
+        return res.status(200).send(user)
+    } catch (err) {
+        return res.status(500).json({message: 'Unable to get user'});
+    }
+}
+

```

Comment:
No query to the DB to check whether the user can be found?

Reply:
This function is just to obtain username from the JWT tokens, login function will check if user can be found.